### PR TITLE
Inspector fix parenting

### PR DESCRIPTION
**Pull request by carolhmj**
Fix a bad change from #13893

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13915/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13915/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13915/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13915/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13915/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13915/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13915/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13915/merge#BCU1XR#0

File: packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/parentPropertyGridComponent.tsx [hunk 1]
```diff
@@ -34,9 +33,8 @@ export class ParentPropertyGridComponent extends React.Component<IParentProperty
         const sortedNodes = scene
             .getNodes()
             .filter((n) => n !== node)
-            .map((n) => this._getNameForSorting(n))
-            .sort((aName, bName) => {
-                return aName.localeCompare(bName);
+            .sort((a, b) => {
```
Comment (Popov72) [inline packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/parentPropertyGridComponent.tsx:36]: I'm not sure why the previous code was not ok?

Comment (carolhmj) [reply packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/parentPropertyGridComponent.tsx:36]: `map` was transforming it in a string array, but the code needed a Node array

**Review by Popov72 — APPROVED** [review]

**Pull request merged**